projects/packages/videopress/src/client/utils/generate-chapters-file/index.ts

```diff
@@ -37,6 +39,15 @@ function generateChaptersFileContent(
 	}
 
 	let content = 'WEBVTT\n';
+
+	// Add a tag to identify auto-generated chapters
```

Comment:
Could we prevent doing this by setting some metadata on the video to say that the chapters were autogenerated? I like this file-change approach (the info is embed on the file itself), just want to make sure we explored other options :)

Reply:
Thanks, Luiz. We wanted to tackle this by adding a prefix to the vtt filename when it's auto-generated, but the API doesn't support setting it.
Using metadata sounds good to me. Reading the file seems more confident, but I don't believe in the best approach.
Maybe we can try it in a follow-up task.

Reply:
I agree, and I actually like the actual approach, it's self contained.